.env.example

```diff
@@ -3,4 +3,7 @@ SENTRY_PROJECT=
 SENTRY_AUTH_TOKEN=
 SENTRY_DSN=
 DOMAIN_FORMAT=
-
+APP_BUILD_VERSION=1
+APP_BINARY_VERSION=dev
+PROJECT_ID=
+NPM_TOKEN=
```

Comment:
without this env, the first line of .npmrc will cause local npm commands to fail (unless the dev logs in with the token already)